Create app fails with typescript dependency

### Steps to reproduce

Running npx feathers --version fails with a message : 
`npm ERR! could not determine executable to run`


Running npm create feathers@pre test fails with a missing typescript dependency:

```
npm create feathers@pre test
node:internal/modules/cjs/loader:998
  throw err;
  ^

Error: Cannot find module 'typescript'
Require stack:
- /home/mrobst/.npm/_npx/92cf463de208346b/node_modules/@feathersjs/generators/lib/commons.js
- /home/mrobst/.npm/_npx/92cf463de208346b/node_modules/@feathersjs/generators/lib/index.js
- /home/mrobst/.npm/_npx/92cf463de208346b/node_modules/@feathersjs/cli/lib/index.js
- /home/mrobst/.npm/_npx/92cf463de208346b/node_modules/create-feathers/bin/create-feathers
    at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/home/mrobst/.npm/_npx/92cf463de208346b/node_modules/@feathersjs/generators/lib/commons.js:34:25)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/mrobst/.npm/_npx/92cf463de208346b/node_modules/@feathersjs/generators/lib/commons.js',
    '/home/mrobst/.npm/_npx/92cf463de208346b/node_modules/@feathersjs/generators/lib/index.js',
    '/home/mrobst/.npm/_npx/92cf463de208346b/node_modules/@feathersjs/cli/lib/index.js',
    '/home/mrobst/.npm/_npx/92cf463de208346b/node_modules/create-feathers/bin/create-feathers'
  ]
}

Node.js v18.12.1

```


### Expected behavior

Feathers app should be created (questions from app generator should appear)

### Actual behavior

Errors as above

### System configuration

```
node -v
v18.12.1

npm -v
8.19.2

uname -a
Linux FIT-L004 5.15.79.1-microsoft-standard-WSL2 #1 SMP Wed Nov 23 01:01:46 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
mrobst@FIT-L004:~/bin>
```

(Per discussion on discord https://discord.com/channels/509848480760725514/930352418179391528/1067829562081288232 )

--- a/packages/generators/package.json
+++ b/packages/generators/package.json
@@ -55,7 +55,8 @@
     "@feathershq/pinion": "^0.3.5",
     "chalk": "^4.0.1",
     "lodash": "^4.17.21",
-    "prettier": "^2.8.1"
+    "prettier": "^2.8.1",
+    "typescript": "^4.9.4"
   },
   "devDependencies": {
     "@feathersjs/adapter-commons": "^5.0.0-pre.35",
@@ -82,8 +83,7 @@
     "mocha": "^10.2.0",
     "shx": "^0.3.4",
     "ts-node": "^10.9.1",
-    "type-fest": "^3.5.0",
-    "typescript": "^4.9.4"
+    "type-fest": "^3.5.0"
   },
   "gitHead": "c641598d9a4de3ceda10f56cf2af288a4236b15e"
 }

